corehq/apps/hqwebapp/static/hqwebapp/less/_hq/modals.less

```diff
@@ -125,3 +125,84 @@
         transform: translate(0, 0) !important;
     }
 }
+
+.add-new-module-modal {
+
+    .modal-header {
+        border-bottom: 1px solid lightgray;
+    }
+
+    .modal-footer {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        margin-top: -20px;
+        margin-bottom:20px;
+        color: #676B70;
+    }
+
+    .modal-flex-container {
+        max-height: 400px;
+        // Make the container scrollable
+        overflow-y: auto;
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        justify-content: flex-start;
+        padding-top: 10px;
+        align-items: stretch;
+    }
+
+    .new-module-option {
+        display: flex;
+        flex-direction: column;
+        // 80px is the total width of the margin
+        width: calc((100% - 80px) / 2);
+        box-sizing: border-box;
+        background-color: transparent;
+        border: none;
+        color: lighten(@cc-text, 20);
```

Comment:
please leave a comment to use one of the available predefined shades in bootstrap5 when migrating